### ISSUE TITLE
Violation of my rules!

### ISSUE THREAD

**Issue by Free-IPTV** [alirezaaziz/deu#1]
Hi @alirezaaziz,

You have copied complete entries of our m3u's, you can see well with all streams with the IP 51.210.199.x.

The ID (link) for tvg-logo = is generated when the link for the graphic is created on the imgur homepage.

This link is an UNIQUE link and is only generated once, it never appears again!!!

You are even so three, you have now created your own repo, in which all the stolen entries can also be found!

If you are not able to create something yourself here at Github, but only to copy it stupidly from others, you are at the wrong place here at Github!

Finally read the Readme in my Repo carefully and write it behind your ears!!! 

https://github.com/Free-IPTV/Countries/blob/master/README.md

If I catch you here on Github again that you are experiencing something without my permission, I promise you a shit storm and you will be reported on Github! Also you copy and paste by others!!!

https://github.com/iptv-org/iptv/pull/2282

**Comment by Zhoro-111** [comment]
Yeah such a liar @alirezaazis,you help nobody stealing from others work and efforts.
SHAME ON YOU!!!
